arch/xtensa/src/esp32/Kconfig

```diff
@@ -665,4 +671,21 @@ config ESP32_WIFI_FS_MOUNTPT
 
 endmenu # ESP32_WIRELESS
 
+menu "Real-Time Timer"
+	depends on ESP32_RT_TIMER
+
+config ESP32_RT_TIMER_TASK_NAME
+	string "Timer task name"
+	default "rt_timer"
+
+config ESP32_RT_TIMER_TASK_PRIORITY
+	int "Timer task priority"
+	default 10
```

Comment:
I find this default value a bit misleading for a "high priority" task.

Reply:
Yes, it is not suitable. I see the high priority workqueue's thread priority is 224 default, far away from max value 255. Is there document introduce exiting task's priority in NuttX.

Reply:
I'm not aware of such document, but I was expecting something higher.  For examples in apps/ the default priority is 100.
So I was expecting to see something between 100 and HP workqueue priority.

Reply:
Yes I think it should be only a little lower than HP workqueue, but it also should process real-time.